src/styleguide/_patterns/01-atoms/01-block-text/05-ascii-art.mustache

```diff
@@ -0,0 +1 @@
+<pre class="grav-c-ascii-art" aria-hidden="true">(╯°Д°)╯︵ ┻━┻</pre>
```

Comment:
the ascii art should probably be extracted in a variable in the `_data.json` file